tools/src/publish-packages/tasks/selectPackagesToPublish.ts

```diff
@@ -55,83 +55,96 @@ async function selectPackageToPublishAsync(
-      message: `Do you want to publish ${green.bold(packageName)} as ${cyan.bold(version!)}?`,
+      message: `Do you want to publish ${green.bold(packageName)} as ${cyan.bold(releaseVersion)}?`,
       default: true,
     },
   ]);
-  if (!selected) {
-    const incrementedVersions = incrementVersion(
-      parcel.pkg.packageVersion,
-      parcel.pkgView?.versions ?? [],
-      options.prerelease === true ? 'rc' : options.prerelease || null
-    );
-    const { version, customVersion } = await inquirer.prompt([
-      {
-        type: 'list',
-        name: 'version',
-        prefix: '❔',
-        message: `What do you want to do with ${green.bold(packageName)}?`,
-        choices: [
-          {
-            name: "Don't publish",
-            value: null,
-          },
-          ...Object.keys(incrementedVersions).map((type) => {
-            return {
-              name: `Publish as ${cyan.bold(incrementedVersions[type])} (${type})`,
-              value: incrementedVersions[type],
-            };
-          }),
-          {
-            name: 'Publish as custom version',
-            value: CUSTOM_VERSION_CHOICE_VALUE,
-          },
-        ],
-        validate: validateVersion(parcel),
-      },
-      {
-        type: 'input',
-        name: 'customVersion',
-        prefix: '❔',
-        message: 'Type in custom version to publish:',
-        when(answers: Record<string, string>): boolean {
-          return answers.version === CUSTOM_VERSION_CHOICE_VALUE;
+
+  if (selected) {
+    return true;
+  }
+
+  const suggestedVersions = getSuggestedVersions(
+    parcel.pkg.packageVersion,
+    parcel.pkgView?.versions ?? [],
+    options.prerelease === true ? 'rc' : options.prerelease || null
+  );
+  const { version, customVersion } = await inquirer.prompt([
+    {
+      type: 'list',
+      name: 'version',
+      prefix: '❔',
+      message: `What do you want to do with ${green.bold(packageName)}?`,
+      choices: [
+        {
+          name: "Don't publish",
+          value: null,
         },
-        validate: validateVersion(parcel),
+        ...suggestedVersions.map((version) => {
+          return {
+            name: `Publish as ${cyan.bold(version)}`,
+            value: version,
+          };
+        }),
+        {
+          name: 'Publish as custom version',
+          value: CUSTOM_VERSION_CHOICE_VALUE,
+        },
+      ],
+      validate: validateVersion(parcel),
+    },
+    {
+      type: 'input',
+      name: 'customVersion',
+      prefix: '❔',
+      message: 'Type in custom version to publish:',
+      when(answers: Record<string, string>): boolean {
+        return answers.version === CUSTOM_VERSION_CHOICE_VALUE;
       },
-    ]);
-    if (customVersion || version) {
-      parcel.state.releaseVersion = customVersion || version;
-      return true;
-    }
+      validate: validateVersion(parcel),
+    },
+  ]);
+
+  if (customVersion || version) {
+    parcel.state.releaseVersion = customVersion || version;
```

Comment:
```suggestion
    parcel.state.releaseVersion = customVersion ?? version;
```
even through this is old code, maybe this is more semantics right.